test/lib/common.bash

```diff
@@ -269,7 +269,9 @@ checkRemovedPackageDB() {
 		for tarch in ${tarches[@]}; do
 			if [ -r "${FTP_BASE}/${repo}/os/${tarch}/${repo}${db%.tar.*}" ]; then
 				for pkgname in ${pkgnames[@]}; do
-					echo "$(bsdtar -xf "${FTP_BASE}/${repo}/os/${tarch}/${repo}${db%.tar.*}" -O)" | grep -qv ${pkgname}
+					if bsdtar -xf "${FTP_BASE}/${repo}/os/${tarch}/${repo}${db%.tar.*}" -O | grep ${pkgname} &>/dev/null; then
+						return 1
+					fi
```

Comment:
grep -v has been converted to if grep; then return 1

I understand why that has been done here, as AFAICT this used to always return successfully since there was always some line that did not contain ${pkgname} -- even if that line was `%NAME%` :D 

Which means we weren't actually testing anything with checkRemovedPackageDB as we cannot actually get test failures.

That being said, this does need to be noted in a commit message.

I also think it would make more sense to check for a pkgname using bsdtar -tf whereas in the other change we need to use -xf in order to check a pkgfile. (I'm not positive why we need to check the pkgfile there either.)